src/libs/actions/PersonalDetails.js

```diff
@@ -84,14 +91,16 @@ function getDisplayName(login, personalDetail) {
 function formatPersonalDetails(personalDetailsList) {
     return _.reduce(personalDetailsList, (finalObject, personalDetailsResponse, login) => {
         // Form the details into something that has all the data in an easy to use format.
-        const avatarURL = getAvatar(personalDetailsResponse, login);
+        const avatar = getAvatar(personalDetailsResponse, login);
         const displayName = getDisplayName(login, personalDetailsResponse);
+
         return {
             ...finalObject,
             [login]: {
                 login,
-                avatarURL,
+                avatar,
                 displayName,
+                pronouns: personalDetailsResponse.pronouns ?? '',
```

Comment:
I think we are just using `||` for now. I don't recall anyone bringing up whether we should use the null coalescing operator or not in JS.

Reply:
Also, are we sure that `personalDetailsResponse` will always at least be an empty object? Otherwise this will throw a `TypeError`. Asking because I see checks for `getAvatar()` and the value is not always assumed to be there.

https://github.com/Expensify/Expensify.cash/blob/64349672d6fde67d2251de791d7694c0be91b784/src/libs/actions/PersonalDetails.js#L44

Reply:
Updated to use `lodashGet` so that we'll return `""` if `personalDetailsResponse` is undefined.